python/hidet/runtime/compiled_graph.py

```diff
@@ -416,6 +416,10 @@ def _save_under(dir_path: str, dir_in_zip: str, exclude: Optional[List[str]] = N
             ge_bytes = json.dumps(asdict(model.graph_execution), indent=4).encode('utf-8')
             f.write(ge_bytes)
 
+        # save dispatch table file
+        if hidet.option.get_store_dispatch_table() and os.path.exists(model.dispatch_table_path):
+            zf.write(model.dispatch_table_path, arcname="dispatch_table.txt")
```

Comment:
Let's keep the way of adding dispatch table to zip file consistent with writing other files like `graph_string.txt` and `graph_execution.json`.